protected/views/patient/view.php

```diff
@@ -92,6 +92,7 @@
 				</section>
 			<?php endif ?>
 			<?php $this->renderPartial('_patient_details')?>
+      <?php $this->renderModulePartials('patient_summary_column1_before_contact')?>
```

Comment:
Alignment looks a little off here.

Reply:
It's because this line uses spaces, while the other lines use tabs.